### PR TITLE
Swagger is no longer a limitation in the Delivery API - it's a feature

### PR DESCRIPTION
**Pull request by kjac**
## Description

Moved the Swagger section from "Limitations" to the "Endpoints" part of the article. The Swagger docs are no longer limited, they should be treated as a feature 😄 

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

V12

## Deadline (if relevant)

Not relevant. As soon as possible, but absolutely no rush 😆

### CODE REVIEW

**Review by elit0451 — APPROVED** [review]
🙌 🙌

File: 12/umbraco-cms/reference/content-delivery-api/README.md [hunk 1]
```diff
@@ -562,6 +562,8 @@ Which properties to expand and therefore include in the output if they refer to
 {% endswagger-response %}
 {% endswagger %}
 
+All endpoints are documented in a Swagger document at `{yourdomain}/umbraco/swagger`. Keep in mind that this document is not available in production mode by default (see also [this article](https://docs.umbraco.com/umbraco-cms/reference/api-versioning-and-openapi)).
```
Comment (alina-tincas) [inline 12/umbraco-cms/reference/content-delivery-api/README.md:565]: When sharing a link it should be described 😁 
```suggestion
All endpoints are documented in a Swagger document at `{yourdomain}/umbraco/swagger`. Keep in mind that this document is not available in production mode by default. For more information check the [API versioning and OpenAPI](https://docs.umbraco.com/umbraco-cms/reference/api-versioning-and-openapi) article.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @kjac thank you for the PR 🙌 

I have just one suggestion, can you please take a look at it? 😊

**Comment by kjac** [comment]
Good stuff @alina-tincas 👍

**Comment by alina-tincas** [comment]
Looks good now so am merging it in 😁

**Pull request merged**